code/modules/research/xenobiology/crossbreeding/_status_effects.dm

```diff
@@ -847,48 +835,74 @@
 /datum/status_effect/stabilized/pink
 	id = "stabilizedpink"
 	colour = "pink"
+	/// List of weakrefs to mobs we have pacified
 	var/list/mobs = list()
-	var/faction_name
+	/// Name of our faction
+	var/faction_name = ""
 
 /datum/status_effect/stabilized/pink/on_apply()
-	faction_name = REF(owner)
+	faction_name = "pink_[REF(owner)]"
```

Comment:
i dunno why but bringing back raw string factions after we just got through with making them all defines (finally) feels weird, especially if someone wants to extend how pink slime factions are generated concurrently and we run into a mismatch and... bleaugh... Here's a small idea of what we _could_ do, I just remain paranoid about people flubbing some minor detail and breaking such nice frameworks.

In the defines file for factions, we could create something like...
```dm
/// Adds ourselves to a Pink Slime Faction specifically tied to whoever brought us into this world.
#define FACTION_PINK(owner) "pink_[owner]"
```

Then in theory, you should be able to invoke this with `FACTION_PINK(REF(owner))`, and the entire framework would be set from there... would almost certainly ensure that we don't have to do more work untangling a potential mess later down the road. Less fragility makes me slightly happier, and we could always re-invoke the macro for finer faction checking if it ever matters (it will likely never matter but I remain eternally paranoid over what coders will do)

if you can think of a more elegant way to leverage define macros that'd be super neat, just throwing this out there.

Reply:
Makes sense to me.